src/libraries/System.Private.CoreLib/src/System/Environment.Browser.cs

```diff
@@ -22,5 +22,7 @@ private static OperatingSystem GetOSVersion()
         {
             return new OperatingSystem(PlatformID.Other, new Version(1, 0, 0, 0));
         }
+
+        private static int GetCurrentProcessId() => (int)Interop.GetCurrentProcessId();
```

Comment:
This is the one and only place `Interop.GetCurrentProcessId` is used. Hardcode 42 here instead of going through extra layer of indirection?

It would match the hardcoding that we are doing for the other properties above. The other properties are not going through extra layer of indirection either.

Reply:
Good suggestion (not sure why I missed that)

Reply:
@jkotas looking at the build failure I know now why it was done this way. There are direct calls to Interop.GetCurrentProcessId within `Microsoft.Diagnostics.Tracing.EventSource` project. Are you ok with revert of the last commit?

Reply:
I have pushed commit that should fix the EventSource build break